src/app/frontend/secret/detail/info.html

```diff
@@ -19,5 +19,10 @@
   <kd-info-card-section>
     <kd-object-meta-info-card object-meta="::$ctrl.secret.objectMeta">
     </kd-object-meta-info-card>
+    <kd-info-card-entry title="[[Type|Secret info details section type entry.]]">
+      <kd-middle-ellipsis display-string="{{::$ctrl.secret.type}}">
+      </kd-middle-ellipsis>
+      <div ng-hide="::$ctrl.secret.type">-</div>
```

Comment:
I do not understand your point at all. You have posted Kubernetes Secret definition, not one from our backend.